package.json

```diff
@@ -189,7 +189,7 @@
     "resin-compose-parse": "^2.1.0",
     "resin-doodles": "0.0.1",
     "resin-image-fs": "^5.0.8",
-    "resin-multibuild": "^3.1.0",
+    "resin-multibuild": "4.0.1-fix-secret-map-filter-ef4d93753d289aa5e56cd4a8bc8538505416d546",
```

Comment:
Pending merge of `resin-multibuild` PR: https://github.com/balena-io-modules/resin-multibuild/pull/55